app/build.gradle

```diff
@@ -235,6 +235,9 @@ dependencies {
     implementation libs.profileinstaller
     baselineProfile project(path: ':baselineprofile')
 
+    debugImplementation libs.chucker.debug
+    releaseImplementation libs.chucker.release
```

Comment:
Inspecting the network traffic is fine on a debug version of the app but not on a release version. Let's keep this only for debug

Reply:
The release version isolates chucker from the release builds that's why its needed

Reply:
I don't understand what you mean by "isolates"?

Users of the production release should not see any debuggable information. This is the same approach with leakcanary. The `release` version is not debuggable

Reply:
I mean the debuggable information will not be available for the release builds that's why is included

Reply:
I checked the "Getting started" section documentation and I see they insist on having both `debug` and `release` defined in the dependencies. I will need to see its behavior on a release build so I am certain the production users are not presented with any debuggable UI

Reply:
> I mean the debuggable information will not be available for the release builds that's why is included

Got you. Going to need to test this on a release build first

Reply:
No problem